frontend/src/features/public-form/components/PublicSwitchEnvMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text } from '@chakra-ui/react'
+
+import { UiCookieValues } from '~shared/types'
+
+import { publicChooseEnvironment } from '~services/EnvService'
+import InlineMessage from '~components/InlineMessage'
+import Link from '~components/Link'
+
+export const PublicSwitchEnvMessage = (): JSX.Element => {
+  const switchToUi = UiCookieValues.Angular
+
+  const handleClick = async () => {
+    await publicChooseEnvironment(switchToUi)
+    window.location.reload()
+  }
+
+  return (
+    <InlineMessage variant="warning" mb="1.5rem" mt={{ base: '2rem', md: '0' }}>
+      <Text>
+        You’re filling this form on the new FormSG. If you have trouble
+        submitting,{' '}
+        <Link onClick={handleClick}>switch to the original one here.</Link>
```

Comment:
semantically, this should be a button (that looks like a link). Links should be used only for navigation.

```tsx
<Button variant='link' onClick={handleClick}>...</Button>
```